### PR TITLE
Add French classification dataset

### PR DESCRIPTION
**Pull request by mmhamdy**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

This PR adds the French [Allocine](https://huggingface.co/datasets/allocine) dataset to the classification tasks.

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

File: mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        date=("2006-01-01", "2020-01-01"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="MIT",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py:32]: Seems like the repo has a suggested citation for the repo. I would use that (in bibtex format)

**Review by KennethEnevoldsen — APPROVED** [review]
Looks good! If you add the points for the dataset and review we can merge it in.

File: mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py (hunk 1 above)
Comment (mmhamdy) [reply mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py:32]: Sure. For the points, 2+4 (for new language task) and 2 for review? just to make sure.

Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py:32]: yep!

Comment (imenelydiaker) [reply mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py]: Hey we already have French classification tasks, so it's not the first task

File: docs/mmteb/points.md [hunk 2]
```diff
@@ -28,7 +28,7 @@
 | cassanof          |              | 6           |          |                     | 1          | 1              |           |                |          |               |
 | taidnguyen        |              | 14          |          |                     |            |                |           |                |          |               |
 | orionw            |              |             |          |                     |      14    |                |           |                |          |               |
-| mmhamdy           |              | 12          |          |                     |            |                |           |                |          |               |
+| mmhamdy           |              | 18          |          |                     |            |                |           |                |          |               |
```
Comment (imenelydiaker) [inline docs/mmteb/points.md:31]:
```suggestion
| mmhamdy           |              | 14          |          |                     |            |                |           |                |          |               |
```

Comment (imenelydiaker) [reply docs/mmteb/points.md:31]: Only +2 because it's not the first task for French

**Review by imenelydiaker — CHANGES_REQUESTED** [review]

**Review by imenelydiaker — APPROVED** [review]
Merging?

File: mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py]: I believe the only one is the translated MassiveIntent?

**Pull request merged**